aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +69,30 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices, const int64_t size) {
+  int64_t numel = crow_indices.numel();
+  auto crow_indices_ = crow_indices.expect_contiguous();
+  auto col_indices_ = col_indices.expect_contiguous();
+  const input_t* crow_indices_data_in = crow_indices_->data_ptr<input_t>();
+  const input_t* col_indices_data_in = col_indices_->data_ptr<input_t>();
+  TORCH_INTERNAL_ASSERT(result.is_contiguous());
+  output_t* data_out = result.data_ptr<output_t>();
+
+  if (numel == 0) {
+    result.zero_();
+    return;
+  }
```

Comment:
nit: Suggest moving this code up just after the line `int64_t numel = crow_indices.numel();`.

Reply:
Done.